dockstore-yaml-proposals/jbrowse/.dockstore.yml

```diff
@@ -0,0 +1,65 @@
+# .dockstore.yml for JBrowse
+version: 1.1
+service:
+  type: docker-compose
+  name: JBrowse
+  author:
+  description: |
+    JBrowse is a fast, scalable genome browser built completely with JavaScript and HTML5.
+    It can run on your desktop, or be embedded in your website.
+  # These are files the Dockstore will index. They will be directly downloadable from Dockstore
+  files:
+     - docker-compose.yml
+     - healthcheck.sh
+     - port.sh
+     - prepare_refdata.sh
+     - stand_up.sh
+     - stop.sh
+     - GRCh37.json
+
+  scripts:
+    # The scripts preprovision, prestart, start, poststart, and postprovision will be executed in order.
+    # The only script required to have a value is start.
+
+    # Before any provisioning
+    preprovision:
+
+    # After provisioning, before starting the service
+    prestart: prepare_refdata.sh
+
+    # Starts the service
+    start: stand_up.sh
+
+    # After the service has started
+    poststart:
+
+    # After the service has started. For multiple provisionings, e.g., the user may decide to add more data later.
+    postprovision:
+
+    # Returns the port the service is exposing. Can be used by a client to know what port to proxy
+    port: port.sh
+
+    # Returns an exit code of 0 if service is running
+    healthcheck: healthcheck.sh
+
+    # Stops the service
+    stop: stop.sh
+
+  environment:
+    JBROWSE_NAME:
+        default: jbrowse
```

Comment:
why is there no quotes on this one but there are quotes below?

Reply:
Because the other one gets treated as a number without the quotes, which when I read it in from Dockstore CLI, became a floating point, which doesn't work as a good value for the port.

Will try to clarify in documentation.

Reply:
I think this should be fixed via code in that we force interpretation of the port to the a fixed point. It should be handled by validation instead of expecting the client to use expectant grammar. I think we should be able to avoid quotes since we are doing some pretty plain YAML.

Reply:
Changed to not require quotes.